data_structures/linked_list.cpp

```diff
@@ -66,11 +52,14 @@ bool isDigit(const std::string &s) {
 class link
 {
 	public:
-		explicit link(int v=0, link* s=nullptr)
-		:val(v),succ(s)
+		explicit link(int v=0, link* s=nullptr) // v and s are used to initialize members value and succ respectively
```

Comment:
please use the function documentation format as mentioned in the CONTRIBUTIONS.md and as has been used for functions elsewhere in the code. 
```
/**
 * description
 * @param v input value for ...
 * @param s is the .....
 */